docs/framework/guides/contributing/plugin-metadata.md

```diff
@@ -0,0 +1,33 @@
+---
+category: framework-contributing
+order: 40
+---
+
+# Plugin metadata
+
+Most of CKEditor 5 packages contain a separate `ckeditor5-metadata.json` file in their root directory. This file contains the basic information about plugins that are exported by the package. Only the main ones that put the given functionality together are described in `ckeditor5-metadata.json`. For example, the `ckeditor5-metadata.json` in the `ckeditor5-mention` package does not contain the `MentionEditing` and `MentionUI` plugins, but instead it only contains the metadata for the main `Mention` plugin, which is a "glue plugin", that loads the editing and UI features.
```

Comment:
This is a nice part of another guide. But in this guide that is located in the `contributing` section we should rather mention that this file can be written by developers that want to make integrations with their plugins easier.

Reply:
I rewrote this section a bit. I think that such a basic introduction with a description of what is the metadata, is really needed. I do not know what else I could add here 🙄